src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx

```diff
@@ -629,6 +672,7 @@ namespace xclhwemhal2 {
           mLogStream << __func__ << " UNZIP of sim bin ended and permissions operation is complete" << std::endl;
 
         simulatorType = getSimulatorType(binaryDirectory);
+        std::transform(simulatorType.begin(), simulatorType.end(), simulatorType.begin(), ::tolower);
```

Comment:
Do not use random C code in C++
```suggestion
        std::transform(simulatorType.begin(), simulatorType.end(), simulatorType.begin(), std::tolower);
```
and add `#include <cctype>` (while sorting the header includes...).